bin/ncp-update-nc

```diff
@@ -186,6 +186,12 @@ $ncc | grep -q db:add-missing-columns && $ncc db:add-missing-columns -n
 $ncc | grep -q db:add-missing-primary-keys && $ncc db:add-missing-primary-keys -n
 $ncc | grep -q db:convert-filecache-bigint && $ncc db:convert-filecache-bigint -n
 
+# Make sure, the standard for trusted domains is used and notify_push is enabled if possible
+nc_domain="$(ncc config:system:get overwrite.cli.url)"
+nc_domain="${nc_domain#http?://*}"
+nc_domain="${nc_domain%%/*}"
+set-nc-domain "${nc_domain}"
```

Comment:
I think it's a good place for it. We want to make sure it won't fail for older versions without notify_push. Also, we want to install notify_push automatically during the upgrade process

Reply:
It doesn't fail for older versions

Reply:
That's also why I used overwrite.cli.url here instead of trusted_domain 3, because earlier versions didn't use the new consistent assignment of trusted_domain slots

Reply:
heads up, this does fail when we have `https://localhost`

```
+ ncc config:system:get overwrite.cli.url
+ nc_domain=http://localhost
+ nc_domain=http://localhost
+ nc_domain=http:
+ set-nc-domain http:
+ local domain=http:
+ local url=https://http:
+ [[ '' == \-\-\n\o\-\t\r\u\s\t\e\d\-\d\o\m\a\i\n ]]
+ ncc config:system:set trusted_domains 3 --value=http:
System config value trusted_domains => 3 set to string http:
+ ncc config:system:set overwrite.cli.url --value=https://http:/
System config value overwrite.cli.url set to string https://http:/
```

Reply:
fixed it